docs/source/commands/apply.rst

```diff
@@ -0,0 +1,32 @@
+Armada - Apply
+==============
+
+
+Commands
+--------
+
+.. code:: bash
+
+    Usage: armada apply FILE
+
+
+    Options:
+
+    [-h] [--dry-run] [--debug-logging] [--disable-update-pre]
+    [--disable-update-post] [--enable-chart-cleanup] [--wait]
+    [--timeout TIMEOUT]
+
+
+Synopsis
+--------
+
+The apply command will consume an armada manifest which contains group of charts
+that it will deploy into the tiller service in your kubernetes cluster.
+Executing the ``armada apply`` again on existing armada deployement will start
+an update of the armada deployed charts.
+
+``amada apply armada-manifest.yaml [--debug-logging]``
+
+If you remove charts from the armada manifest and exectute
+``--enable-chart-cleanup``. Armada will remove any charts with the armada
```

Comment:
could you explain how this works in more detail? especially with https://github.com/att-comdev/armada/pull/109 it's very unclear to me how this would work in practice - as it would seem impossible to reliably target the appropriate charts if Amada is managing more than a single deployment.